src/deploy/table_account_preference_event_size_policy.sql

```diff
@@ -0,0 +1,24 @@
+-- Deploy maevsi:table_account_preference_event_size_policy to pg
```

Comment:
A few `requires:` see to be missing here

Reply:
We can also get rid of all these comments as they add more complexity than they help, I now notice. Let's do this in a different PR though.

Reply:
Done